render/jpql/src/main/kotlin/com/linecorp/kotlinjdsl/render/jpql/JpqlRenderContext.kt

```diff
@@ -287,6 +288,7 @@ private class DefaultModule : JpqlRenderModule {
             JpqlEqualAnySerializer(),
             JpqlEqualSerializer(),
             JpqlExistsSerializer(),
+            JpqlExpSerializer(),
```

Comment:
It seems to me that this code should be placed under JpqlExpressionSerializer(). If you look at it in alphabetical order

Reply:
Ah, I have followed import line order :)